homeassistant/components/media_player/squeezebox.py

```diff
@@ -95,7 +95,8 @@ def create_players(self):
         """Create a list of devices connected to LMS."""
         result = []
         data = yield from self.async_query('players', 'status')
-
+        if data is False:
+            return
```

Comment:
I think `return result` is better. You should not call async_add_devices with None.